### ISSUE TITLE
Suggestion: refactor startup script 999-load.py as a method

### ISSUE THREAD

**Issue by danielballan** [xpdAcq/xpdAn#29]
If I understand correctly, you distribute 999-load.py as a script that is meant to be included in an IPython profile. This makes it difficult to manage different versions on xpdAn (or xpdAcq) on a single account. I would suggest moving the code into a top-level function, `xpdAn.setup()`. All versions of xpdAn/xpdAcq can have this functions (it could be backported to previous releases). Then, the IPython profile can safely call `xpdAn.setup()` and `xpdAcq.setup()` -- possibly taking as arguments any local variables it needs, such as `pe1`. The makes it possible to upgrade xpdAn/xpdAnl without editing the profile.

**Comment by CJ-Wright** [comment]
Will the function correctly import things into the session namespace?

**Comment by danielballan** [comment]
Good question. It is possible to use a function to add things to the global namespace, but this is generally discouraged (and hard to follow). I would say that should probably be done separately. E.g.,:

```python
# earlier, in XPD's IPython profile
pe1 = ...
```

```python
# 999-load.py
from xpdAn import useful_function1, useful_function2, setup
xpdAn.setup(pe1=pe1)
```

**Comment by CJ-Wright** [comment]
Awesome!

**Comment by sbillinge** [comment]
If this is a good suggestion, I like it.  As with #30 this was something we really struggled with, maintaining multiple versions of startup and so on, so a more elegant solution that works would be awful nice.

Again, let's make a branch for testing for v0.6 or later, since everything is actually working right now.
